MovieShowcaseSPA/MovieShowcaseSPA.csproj

```
@@ -23,7 +23,7 @@
 
 <ItemGroup>
 <PackageReference Include="Microsoft.AspNetCore.SpaProxy" Version="7.0.0" />
-<PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="7.0.0" />
+<PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="7.0.1" />

```

No this is not correct. Please revert this change.
```suggestion
<PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="7.0.0" />

```